tests/dsl/src/test/java/org/mule/test/extension/dsl/AbstractElementModelTestCase.java

```diff
@@ -222,4 +204,8 @@ protected void createAppDocument() throws ParserConfigurationException {
                         "http://www.mulesoft.org/schema/mule/core http://www.mulesoft.org/schema/mule/core/current/mule.xsd http://www.mulesoft.org/schema/mule/tls http://www.mulesoft.org/schema/mule/tls/current/mule-tls.xsd http://www.mulesoft.org/schema/mule/jmsn http://www.mulesoft.org/schema/mule/jmsn/current/mule-jmsn.xsd http://www.mulesoft.org/schema/mule/sockets http://www.mulesoft.org/schema/mule/sockets/current/mule-sockets.xsd http://www.mulesoft.org/schema/mule/db http://www.mulesoft.org/schema/mule/db/current/mule-db.xsd http://www.mulesoft.org/schema/mule/httpn http://www.mulesoft.org/schema/mule/httpn/current/mule-httpn.xsd");
   }
 
+  protected void assertValue(DslElementModel elementModel, String value) {
+    assertThat(elementModel.getValue().get(), is(value));
```

Comment:
test for "get value from element", fixes => https://www.mulesoft.org/jira/browse/MULE-11537